### ISSUE TITLE
Feed event not populating with attributes

### ISSUE THREAD

**Issue by jaxre** [MISP/MISP#3043]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | RHEL 7.4 (Maipo)
| OS version (client)       | Windows 7
| PHP version               | 7.1
| MISP version / git hash   | 04c89499a234da5d6596a711fb65522b5c793a23
| Browser                   | N/A

### Expected behavior
When downloading an external feed, event should populate with attributes.

### Actual behavior
Event is created, but no attributes are added.

### Steps to reproduce the behavior
configure proxy settings
Navigate to List Feeds > Feed # > Click "Cache Feed" in Actions column
When job completes, event gets created but no attributes are added.

### Logs, screenshots, configuration dump...

**Diagnostics page:**
Proxy

This tool tests whether your HTTP proxy settings are correct.
Proxy settings…OK	

**/var/www/MISP/app/tmp/logs/error.log**
2018-03-14 14:00:57 Warning: Invalid argument supplied for foreach() in [/var/www/MISP/app/Model/Feed.php, line 975]

**/var/www/MISP/app/Model/Feed.php, line 975:**
```
        private function __cacheFreetextFeed($feed, $redis, $HttpSocket, $jobId = false) {
                if ($jobId) {
                        $job = ClassRegistry::init('Job');
                        $job->id = $jobId;
                        if (!$job->exists()) {
                                $jobId = false;
                        }
                }
                $values = $this->getFreetextFeed($feed, $HttpSocket, $feed['Feed']['source_format'], 'all');
                if (!empty($values)) {
          975---> foreach ($values as $k => $value) {
                                $redis->sAdd('misp:feed_cache:' . $feed['Feed']['id'], md5($value['value']));
                                $redis->sAdd('misp:feed_cache:combined', md5($value['value']));
                                if ($jobId && ($k % 1000 == 0)) {
                                                $job->saveField('message', 'Feed ' . $feed['Feed']['id'] . ': ' . $k . ' values cached.');
                                }
                        }
                        $redis->set('misp:feed_cache_timestamp:' . $feed['Feed']['id'], time());
                        return true;
                }
                return false;
```

**Configurations:**
OS:
    export http_proxy="http://[username]:[password]@[hostname.domain.com]:[port]"
    export https_proxy=$http_proxy

MISP:
    Proxy.host = http://[hostname.domain.com]
    Proxy.port = [port]
    Proxy.method = Basic
    Proxy.user = [username]
    Proxy.password = [password]

Config.php (Configured via settings in web app)
     'Proxy' =>
         array (
         'host' => '[hostname.domain.com]',
         'port' => [port],
         'method' => 'Basic',
         'user' => '[username]',
         'password' => '[password]',

Feed Settings:
    [X] Enabled
    [X] Lookup Visible
    Name: http://www.cert.at/static/downloads/data/conficker/all_domains.txt
    Provider: cert.at
    Source Format: Network
    URL: http://www.cert.at/static/downloads/data/conficker/all_domains.txt
    Source Format: Freetext (Also tried "Simple CSV parsed feed")
    Target Event: Fixed Event
    Target Event ID: [eventid]
    Exclusion Regex: [empty]
    [ ] Auto Publish
    [ ] Override IDS flag
    [ ] Delta Merge
    Distribution: Your organisation only
    Default Tag: None
    Filter Rules: N/A

**Packet Captures**

(Clicking "Cache feed" in List Feeds > Feed # > Actions Column)
[ip.addr == 83.136.38.146]
649	13.944240	[ip address]	**83.136.38.146**	TCP	74	36874 → 80 [SYN] Seq=0 Win=29200 Len=0 MSS=1460 SACK_PERM=1 TSval=923260381 TSecr=0 WS=128
649	13.944240	[ip address]	**83.136.38.146**	TCP	74	36874 → 80 [SYN] Seq=0 Win=29200 Len=0 MSS=1460 SACK_PERM=1 TSval=923260381 TSecr=0 WS=128
649	13.944240	[ip address]	**83.136.38.146**	TCP	74	36874 → 80 [SYN] Seq=0 Win=29200 Len=0 MSS=1460 SACK_PERM=1 TSval=923260381 TSecr=0 WS=128
649	13.944240	[ip address]	**83.136.38.146**	TCP	74	36874 → 80 [SYN] Seq=0 Win=29200 Len=0 MSS=1460 SACK_PERM=1 TSval=923260381 TSecr=0 WS=128

[ip.addr == [proxy ip]
N/A
N/A
N/A

(pulling with wget)
wget http://www.cert.at/static/downloads/data/conficker/all_domains.txt
[ip.addr == 83.136.38.146]
N/A
N/A
N/A

[ip.addr == [proxy ip]
349	6.203587	[sourceip] 	[proxyip]  	TCP	74	45686 → [proxyport] [SYN] Seq=0 Win=29200 Len=0 MSS=1460 SACK_PERM=1 TSval=924151868 TSecr=0 WS=128
351	6.203919	[sourceip] 	[proxyip]  	TCP	74	[proxyport] → 45686 [SYN, ACK] Seq=0 Ack=1 Win=28960 Len=0 MSS=1460 SACK_PERM=1 TSval=2450609716 TSecr=924151868 WS=4
352	6.203955	[sourceip] 	[proxyip]  	TCP	66	45686 → [proxyport] [ACK] Seq=1 Ack=1 Win=29312 Len=0 TSval=924151869 TSecr=2450609716

**Comment by shanto12** [comment]
i am faing the same issue. Any updates on this please?

**Comment by JakubOnderka** [comment]
This should be fixed in #5052. Or at least error logs will provide more information what happened.
